php/libraries/Candidate.class.inc

```diff
@@ -203,8 +203,8 @@ class Candidate
         ?string $edc,
         string $sex,
         ?string $PSCID = null,
-        int $projectID = null
-    ): int {
+        ?int $projectID = null
```

Comment:
Now that use projects is mandatory, this should not have a `?` however if it is causing you issues, fixing it would be outside of the scope of your PR and should be reported as an issue on github